src/fmi_export/fmiCollector.cpp

```diff
@@ -41,13 +41,6 @@ std::unique_ptr<collector> fmiCollector::clone() const
 void fmiCollector::cloneTo(collector *gr) const
 {
 	collector::cloneTo(gr);
-
-	auto nrec = dynamic_cast<fmiCollector *>(gr);
```

Comment:
Was something supposed to happen to nrec?

Reply:
I don't think the function was finished.  The fmi stuff is still in progress, so some parts are not finished.  so yes but it hasn't been written yes, and I am not sure of a use case for the function yet just do the nature of the situations in which this is used.

Reply:
I am guessing I just put the boilerplate code in place

Reply:
👍